sdk/core/Azure.Core.TestFramework/src/PlaybackOnlyAttribute.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using NUnit.Framework;
+using NUnit.Framework.Interfaces;
+using NUnit.Framework.Internal;
+
+namespace Azure.Core.TestFramework
+{
+    /// <summary>
+    /// Attribute on test assemblies, classes, or methods that run only against playback resources.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class | AttributeTargets.Assembly, AllowMultiple = true, Inherited = true)]
+    public class PlaybackOnlyAttribute : NUnitAttribute, IApplyToTest
```

Comment:
Would this prevent someone from being able to re-record the test just by changing the ctor parameter?

Reply:
Which constructor parameter are you referring to?

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/b9bd85cb2591f7c3f3c054f33f158e758056f1d3/sdk/storage/Azure.Storage.Blobs/tests/BlobUriBuilderTests.cs#L16

Reply:
I assume so, yes, but this is by design.  The workflow to re-record would be to comment out the attribute.